modules/surge-docs/src/test/scala/docs/command/BankAccountEngine.scala

```diff
@@ -8,7 +8,7 @@ import surge.scaladsl.command.SurgeCommand
 
 // #bank_account_engine_class
 object BankAccountEngine {
-  lazy val surgeEngine: SurgeCommand[UUID, BankAccount, BankAccountCommand, Nothing, BankAccountEvent] = {
+  lazy val surgeEngine: SurgeCommand[UUID, BankAccount, BankAccountCommand, BankAccountEvent] = {
```

Comment:
TODO This breaks backwards compatibility but was a bit awkward anyways, take a look at the impact of this change and possibly revert it for now

Reply:
The way things are layered today this breaks compatibility in the open source layers but would not impact users internal to UKG since this particular API is hidden behind one flavored with UKG specific types. If we had these artifacts publicly published we'd want to be more careful of preserving compatibility here, but I think it's fine to break this API here especially while we know of and can limit the exact impact.